packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -721,14 +721,39 @@ class FileCollection extends DocumentCollection {
     return formatBytes(parseInt(file.size), decimal)
   }
 
-  async downloadArchive(fileIds, notSecureFilename = 'files') {
+  /**
+   * Attributes used for create archive link by ids
+   *
+   * @typedef {object} ArchivePages
+   * @property {string} id - Id of the file
+   * @property {number} page - The page number
+   */
+
+  /**
+   * Download an archive of the files
+   *
+   * @param {string[]} fileIds - List of file ids
+   * @param {string} notSecureFilename - Name of the archive
+   * @param {ArchivePages[]} pages - Array of objects, with `id` the file identifier, and `page` the page number (1 is the first page)
+   */
+  async downloadArchive(fileIds, notSecureFilename = 'files', pages) {
```

Comment:
nit: Function signature is maybe a little confusing ? Seems like `notSecureFilename` was not mandatory before but now : 
- it is in jsdoc
- we added a argument behind so it is (even if we can pass undefined)

Reply:
Good point, corrected